### PR TITLE
Hard-code version in installer, remove --url option, rename it

### PR DESCRIPTION
**Pull request by labbati**
### Description

- `--version` removed as it's now version-locked.
- `--url` removed as it does not add any covered use case that would be not covered by `--file`.
- Script renamed based on internal discussion to `datadog-setup.php`.

### Readiness checklist
- [x] (only for Members) Changelog has been added to the release document.
- [x] Tests added for this feature/bug.

### Reviewer checklist
- [x] Appropriate labels assigned.
- [x] Milestone is set.
- [x] Changelog has been added to the release document. For community contributors the reviewer is in charge of this task.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,7 +16,8 @@ ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 PHP_EXTENSION_DIR=$(shell php -r 'print ini_get("extension_dir");')
 PHP_MAJOR_MINOR:=$(shell php -r 'echo PHP_MAJOR_VERSION . PHP_MINOR_VERSION;')
 
-VERSION:=$(shell awk -F\' '/const VERSION/ {print $$2}' < src/DDTrace/Tracer.php)
+VERSION := $(shell awk -F\' '/const VERSION/ {print $$2}' < src/DDTrace/Tracer.php)
+PROFILING_RELEASE_URL := https://github.com/DataDog/dd-prof-php/releases/download/v0.3.0-rc.6/datadog-profiling.tar.gz
```
Comment (morrisonlevi) [inline Makefile:20]: Release 0.3.0 has been made with no changes from the RC6 other than the version number:

```suggestion
PROFILING_RELEASE_URL := https://github.com/DataDog/dd-prof-php/releases/download/v0.3.0/datadog-profiling.tar.gz
```

**Review by morrisonlevi — COMMENTED** [review]
This looks pretty good to me. I'm a bit confused on one detail, though: the generated `datadog-setup-{$platform}.php` files don't differ at all, I think? Is that right?

File: datadog-setup.php [hunk 2]
```diff
@@ -451,17 +451,12 @@ function parse_validate_user_options()
...
 
     if (!$normalizedOptions[OPT_UNINSTALL]) {
-        // Only one of --url and --file may be provided.
-        $installables = array_intersect([OPT_URL, OPT_FILE], array_keys($options));
+        // Only one --file can be provided.
+        $installables = array_intersect([OPT_FILE], array_keys($options));
         if (count($installables) > 1) {
-            print_error_and_exit('Only one of --url and --file must be provided', true);
+            print_error_and_exit('Only one --file must be provided', true);
         }
```
[the remark this answers is not in the document]
Comment (labbati) [reply datadog-setup.php:458]: good catch

**Review by morrisonlevi — APPROVED** [review]

**Review by morrisonlevi — APPROVED** [review]

**Review by SammyK — APPROVED** [review]
LGTM 👍

**Pull request merged**